### PR TITLE
MessageSigner: Return some info about valid signatures

### PR DESCRIPTION
**Pull request by spladug**
This is useful if we want to collect metrics on how old signatures are
when they are used, even if valid.

:eyeglasses: @MelissaCole @umbrae 

cc: WEB-1622

### CODE REVIEW

File: tests/unit/crypto_tests.py [hunk 1]
```diff
@@ -34,6 +34,17 @@ def test_roundtrip(self):
         self.signer.validate_signature(self.message, signature)
 
     @mock.patch("time.time")
+    def test_signature_info(self, time):
+        time.return_value = 0
+        signature = self.signer.make_signature(self.message, max_age=datetime.timedelta(seconds=30))
+
+        time.return_value = 10
```
Comment (MelissaCole) [inline tests/unit/crypto_tests.py]: Why is this line necessary? The expiration won't change as the time changes, so I'm just curious

**Comment by MelissaCole** [comment]
:fish: lgtm

File: tests/unit/crypto_tests.py (hunk 1 above)
Comment (spladug) [inline tests/unit/crypto_tests.py]: iiit's not. good point. removed!

**Comment by spladug** [comment]
f5dd8522c609d8628ca90fedf5cd8b2ce67fe9be
